src/ussd_higherhealth_healthcheck.js

```diff
@@ -402,6 +412,7 @@ go.app = (function () {
         opts.http_error_count = _.get(opts, "http_error_count", 0) + 1;
         if (opts.http_error_count === 3) {
           self.im.log.error(e.message);
+          console.log(e, e.message, '===========================');
```

Comment:
This should be removed, `console` is not accessible while in the sandboxed environment, so the app will crash if it reaches this point